scripts/clone-and-validate-theme-starters.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+CLONE=$1
+IS_CI="${CI:-false}"
+BASE=$(pwd)
+COMMIT_MESSAGE=$(git log -1 --pretty=%B)
+
+if [ "$IS_CI" = true ]; then
+  sudo apt-get update && sudo apt-get install jq
+fi
+
+for folder in "themes/gatsby-starter-blog-theme" "themes/gatsby-starter-notes-theme" "themes/gatsby-starter-theme"; do
```

Comment:
This seems a little less reusable than it was — and I’m not sure why it’s a separate script? Could you share some info?

It seems like the central problem that needs to be solved is iterating only a subset of themes, right?

Reply:
Yeah, only want to go over those three specific folders in the `themes` directory (since there are also two more theme packages that this shouldn't run on) 

Again, didn't bother because will be moving these over to the `packages` and `starters` directories after we're stable and throw this away.